resolvers/index.js

```diff
@@ -137,4 +140,35 @@ module.exports = {
     siteRecord.fields.InventoryUpdates = inventoryUpdates;
     return siteRecord;
   },
+  [Tables.PurchaseRequests]: {
+    write: async (purchaseRequestRecord, authRecord) => {
+      // Only allow admin users to review purchase requests
+      if (
+        purchaseRequestRecord.fields.hasOwnProperty("Status") &&
+        purchaseRequestRecord.fields.Status !== "Pending"
```

Comment:
My only concern with this is that I have status as an enum on the frontend, so if this is ever renamed in Airtable and/or on the frontend, this will break.

Reply:
I think this is just something that cross-server communication has to deal with in general (we need to establish & maintain a contract @ some point). Solution is probably to document well on both frontend and backend (Have a comment that says you need to change the value on the frontend/backend also, etc...)?

This would probably be a neat addition to schema generator, where it'd also establish enums for any "Select" fields.